sky/shell/platform/ios/framework/Source/accessibility_bridge.mm

```diff
@@ -175,6 +175,75 @@ - (NSInteger)indexOfAccessibilityElement:(id)element {
   return (_children == nil) ? NSNotFound : [_children indexOfObject:element];
 }
 
+#pragma mark - UIAccessibilityAction overrides
+
+- (BOOL)accessibilityActivate {
+  // TODO(tvolkert): Implement
+  return NO;
+}
+
+- (void)accessibilityIncrement {
+  // TODO(tvolkert): Implement
+}
+
+- (void)accessibilityDecrement {
+  // TODO(tvolkert): Implement
+}
+
+- (BOOL)accessibilityScroll:(UIAccessibilityScrollDirection)direction {
+  BOOL canBeScrolled;
```

Comment:
Nit: Uninitialized local variable.

Not that it matters in this particular case. But we always initialize our locals :)